ftgo-order-service/src/main/java/net/chrisrichardson/ftgo/orderservice/domain/OrderServiceConfiguration.java

```diff
@@ -38,8 +43,13 @@ public OrderService orderService(RestaurantRepository restaurantRepository, Orde
   }
 
   @Bean
-  public SagaManager<CreateOrderSagaState> createOrderSagaManager(CreateOrderSaga saga) {
-    return new SagaManagerImpl<>(saga);
+  public SagaManager<CreateOrderSagaState> createOrderSagaManager(CreateOrderSaga saga,
```

Comment:
`SagaManager` should not longer be used by applications.

Applications should use `SagaInstanceFactory` - see https://github.com/eventuate-tram/eventuate-tram-sagas/commit/77bd85b978f2e80dd3088a81d061a65f6d465668  and https://github.com/eventuate-tram/eventuate-tram-sagas-examples-customers-and-orders/commit/168cc54a1807abdfc5d0b99a1def9a39c1f36aaf

Reply:
Missed it. Sorry.